components/ui/NavItem.jsx

```diff
@@ -0,0 +1,18 @@
+import Link from "next/link";
+import classNames from "classnames";
+import styles from "./Navbar.module.css";
+
+
+const NavItem=({title,link,isActive,index})=>{
+
+    return(
+        <li
+            className={classNames(styles.navbarLinks, {
```

Comment:
Also here, some indentation would make this file more readable. (see my other comment below)

Reply:
Done